tests/legacy-cli/e2e/tests/basic/scripts-array.ts

```diff
@@ -54,6 +54,7 @@ export default function () {
     // index.html lists the right bundles
     .then(() => expectFileToMatch('dist/test-project/index.html', oneLineTrim`
       <script type="text/javascript" src="runtime.js"></script>
+      <script type="text/javascript" src="es2015-polyfills.js" nomodule></script>
```

Comment:
Drive-by comment: can we drop the `type="text/javascript"` please? It’s redundant: https://mathiasbynens.be/notes/html5-levels#type-attributes

Reply:
Yes.  A refactor and cleanup of the index generation is planned that will also address this.